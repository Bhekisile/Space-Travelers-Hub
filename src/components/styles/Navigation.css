@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@200;400;500;600;700&display=swap');

body {
  font-family: 'Montserrat', sans-serif;
}

.navbar {
  display: flex;
  justify-content: space-between;
  align-items: center;
  height: 100px;
  width: 90%;
  padding-left: 20px;
  padding-right: 46px;
  margin-left: 44px;
  border-bottom: 1px solid rgba(111, 111, 111, 0.6);
}

.nav-left {
  display: flex;
  justify-content: space-between;
  align-items: center;
  width: 420px;
}

.nav-right {
  display: flex;
  width: 370px;
  justify-content: space-between;
}

a {
  text-decoration: none;
  font-size: 16px;
  color: #0290ff;
  padding: 0 16px;
}

a:hover {
  cursor: pointer;
}

.view-profile {
  border-left: 3px solid rgba(111, 111, 111, 0.6);
  padding-left: 30px;
}
