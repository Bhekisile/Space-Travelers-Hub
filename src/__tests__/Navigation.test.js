import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom'; // Import BrowserRouter
import Navigation from '../components/Navigation'; // Adjust the path

test('Navigation component matches snapshot', () => {
  const { container } = render(
    <Router>
      {' '}
      {/* Wrap your component with the Router */}
      <Navigation />
    </Router>,
  );
  expect(container).toMatchSnapshot();
});
