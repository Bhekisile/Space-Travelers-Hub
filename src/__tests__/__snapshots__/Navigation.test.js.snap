// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation component matches snapshot 1`] = `
<div>
   
  <div>
    <div
      class="navbar"
    >
      <div
        class="nav-left"
      >
        <img
          alt=""
          src="planet.png"
          style="width: 60px; height: 60px;"
        />
        <h1>
          Space Travelers' Hub
        </h1>
      </div>
      <div
        class="nav-right"
      />
      <a
        class="view"
        href="/rockets"
      >
        Rockets
      </a>
      <a
        class="view"
        href="/missions"
      >
        Missions
      </a>
      <a
        class="view-profile"
        href="/profile"
      >
        My Profile
      </a>
    </div>
  </div>
</div>
`;
